app/policies/pageflow/entry_policy.rb

```diff
@@ -120,15 +120,17 @@ def update_account_on?
     end
 
     def update_theming_on?
-      publish_on_account_of?
+      Pageflow.config.permissions.show_theming_dropdown_to_non_admin &&
+        publish_on_account_of?
     end
 
     def manage_account_of?
       query.has_at_least_account_role?(:manager)
     end
 
     def update_feature_configuration_on?
-      manage_account_of?
+      Pageflow.config.permissions.show_entry_features_to_non_admin &&
+        manage_account_of?
     end
```

Comment:
These policy changes need specs. Let's add a protected `permissions_config` method to `ApplicationPolicy`:

    def permissions_config
      Pageflow.config.permissions
    end

This way we can avoid duplicating the concrete look up logic.